docs/victoriametrics-cloud/get-started/_index.md

```diff
@@ -11,17 +11,14 @@ menu:
 In this section you will find everything you need to start using [VictoriaMetrics Cloud](https://console.victoriametrics.cloud/signUp?utm_source=website&utm_campaign=docs_vm_get_started).
 
 * [Overview of VictoriaMetrics Cloud](overview/)
+* [Key Features and Benefits](features/)
```

Comment:
please use absolute links :)

Reply:
Thanks. Changing these and other links in the next PR